cesium/internal/unary/writer.go

```diff
@@ -22,14 +22,16 @@ import (
 	"github.com/synnaxlabs/x/control"
 	"github.com/synnaxlabs/x/override"
 	"github.com/synnaxlabs/x/telem"
+	"github.com/synnaxlabs/x/validate"
 )
 
 type WriterConfig struct {
-	Start     telem.TimeStamp
-	End       telem.TimeStamp
-	Subject   control.Subject
-	Authority control.Authority
-	Persist   *bool
+	Start           telem.TimeStamp
+	End             telem.TimeStamp
+	Subject         control.Subject
+	Authority       control.Authority
+	Persist         *bool
+	AutoPersistTime telem.TimeSpan
```

Comment:
Should be `AutoPersistInterval` no?

Reply:
Also I just realized it but this variable is kind of confusing, and doesn't explain that this controls domain persistence and not persistence in general. Something to thing about. Also we should document all of the properties of this struct.